js/deer-record.js

```diff
@@ -369,7 +369,10 @@ export default class DeerReport {
                             if(anno.new_obj_state.creator)input.setAttribute(DEER.ATTRIBUTION, anno.new_obj_state.creator)
                     })
                 })
-            return Promise.all(annotations).then(() => entity)
+            return Promise.all(annotations).then(() => {
+                DEER.UTILS.broadcast(undefined,DEER.EVENTS.UPDATED, this.elem, entity)
```

Comment:
This is throwing an undefined error.

I think you mean ` UTILS.broadcast(undefined,DEER.EVENTS.UPDATED, this.elem, entity)`

Reply:
oops. It ran for me, but I have a bit more around it that may have replicated UTILS.